addon/ng2/blueprints/ng2/files/__path__/test.ts

```diff
@@ -14,6 +14,14 @@ import 'zone.js/dist/sync-test';
 // Unfortunately there's no typing for the `__karma__` variable. Just declare it as any.
 declare var __karma__: any;
 
+// Typings reference file, see links for more information
```

Comment:
These comments are useless. Please provide a comment that explains why we need those declaration here.